docs/SIL.rst

```diff
@@ -6117,6 +6117,29 @@ a type `T` into the move only value space.
 The ``lexical`` attribute specifies that the value corresponds to a local
 variable in the Swift source.
 
+
+drop_deinit
+```````````
+
+::
+
+   sil-instruction ::= 'drop_deinit' sil-operand
+
+   %1 = drop_deinit %0 : $T
+   // T must be a move-only type
+   // %1 is an @owned T
+   %3 = drop_deinit %2 : $*T
```

Comment:
@eeckstein do need to support an address form for `drop_deinit`?
I really hope we don't need that! It's easy to misuse and hard to verify.
In the meantime, can you remove it from the docs?

If we need an address form, we should discuss whether that should be `drop_deinit_addr`. Generally, instructions should have fixed number and type of operands except in special cases. `debug_value` is an exception where it works out ok. But an instruction that produces a new address is not straightforward.